mods/gen1/moves.js

```diff
@@ -107,9 +107,9 @@ exports.BattleMovedex = {
 					return;
 				}
 				let moves = pokemon.moveset;
-				for (let i = 0; i < moves.length; i++) {
-					if (moves[i].id !== 'bide') {
-						pokemon.disableMove(moves[i].id);
+				for (let move of moves) {
```

Comment:
`const`

Reply:
Ditto every other instance of it not being modified.

Reply:
Also `for (const move of pokemon.moveset) {`

No need to cache `moves` when you don't need to access it repeatedly.